### DISCUSSION TITLE
Filtering Dependabot notifications

### DISCUSSION THREAD

**Discussion by slifty** [community/community#48846] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I have a *lot* of dependabot noise in my notifications.

Don't get me wrong I love the little bloop-bleeper, but it makes the notifications interface incredibly noisy and I'd love a way to have a view of my notifications that *excludes* dependabot.

The current documentation says that exclusion is not supported in the filter syntax, so I don't have a clear path for solving this right now.  Is there (or could there be) a way for me to be able to only see notifications generated by human actions?

**Comment by Wulf** [comment]
Yes please! I was just about to create an issue for this. Dependabot is making it harder for me to parse my notifications and address issues/PRs created by others

**Comment by ian-h-chamberlain** [comment]
BTW, there is also a much older discussion #5601 for a more general solution, which would be to support the negative filter syntax. Agreed that filtering out bot notifications would be an important UX improvement.

**Comment by ADTC** [comment]
As a workaround, try this:

1. Create a _positive_ filter to search for all unread dependabot notifications. (one-time setup)
2. Use the filter to find and mark them all as read.
3. Look at the Inbox for the remaining unread notifications.

Since dependabot notifications are now already marked read, they won't show up in the Inbox.
